container.go

```diff
@@ -183,7 +206,37 @@ func (container Container) Resolve(out interface{}, id ...string) {
 }
 
 // Apply wiring to all components.
-func (container Container) Apply() {
+func (container Container) Apply(to ...interface{}) {
+
+	if len(to) > 0 {
+
+		rv := reflect.ValueOf(to[0])
+		rt := rv.Type()
+		_, file, no, _ := runtime.Caller(container.callerSkip + 1)
+
+		comp := component{
+			id:         "",
+			value:      rv,
+			declaredAt: file + ":" + strconv.Itoa(no),
+		}
+
+		if rt.Kind() == reflect.Ptr {
+			rt = rt.Elem()
+			rv = rv.Elem()
+			comp.value = rv
+		}
+
+		comp.dependencies = dependinces(
+			container,
+			rt,
+			rv,
+		)
+
+		container.fill(comp)
+
+		return
```

Comment:
why do we need to return here? this will cause other components that connected using `Connect` to be not added right?